openshift/templates/cakephp-mysql.json

```diff
@@ -354,6 +354,12 @@
   ],
   "parameters": [
     {
+      "name": "NAME",
+      "displayName": "Name",
+      "description": "The name assigned to all of the objects defined in this template.",
```

Comment:
OK, will update to say "frontend objects".

A fair amount of what I would perceive to be db-related objects are already parameterized.

A potential list of things to update with a current setting with "mysql" in it (please confirm/deny):
1) the template itself has a "mysql" annotation tag
2) the Service.spec.port.name 
3) the DC.spec.triggers.imageChangeParams.containerNames
4) the DC.spec.triggers.imageChangeParams.from.name
5) the DC.spec.template.spec.containers.name and image